service/src/test/java/com/ge/predix/acs/privilege/management/dao/GraphSubjectRepositoryTest.java

```diff
@@ -18,12 +34,14 @@
 import org.testng.annotations.Test;
 
 import com.ge.predix.acs.config.GraphConfig;
+import com.ge.predix.acs.model.Attribute;
+import com.ge.predix.acs.rest.Parent;
+import com.ge.predix.acs.utils.JsonUtils;
 import com.ge.predix.acs.zone.management.dao.ZoneEntity;
 import com.thinkaurelius.titan.core.TitanFactory;
 
 public class GraphSubjectRepositoryTest {
```

Comment:
This class should include the hierarchical tests in resource respository test.  We should try to reuse the tests with some parameterization.